### PR TITLE
revert bulk_tags_edit_spec change

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made? 🤔

To fix a mistake I added to dep updates.

### CODE REVIEW

**Review by justinlittman — APPROVED** [review]

File: spec/features/bulk_tags_edit_spec.rb [hunk 1]
```diff
@@ -57,7 +57,7 @@
           wait_for_download
           druids_with_tags = CSV.parse(File.read(download))
           expect(druids_with_tags.count).to eq(number_of_druids)
-          druids_with_tags.each do |druid, tags|
+          druids_with_tags.each do |druid, *tags|
```
Comment (jmartin-sul) [inline spec/features/bulk_tags_edit_spec.rb:60]: the rubocop exception comment needs to be restored too, i think: https://github.com/sul-dlss/infrastructure-integration-test/pull/666/commits/c4edb8e7123ab38cbeab016d5f28ef37b470907c#diff-1413f9d516eee302bb7e789fe56d0135bf7edc823b4dfbf04b63c5ece6188511L60

```suggestion
          druids_with_tags.each do |druid, *tags| # rubocop:disable Style/HashEachMethods rubocop seems mistaken in thinking the tags var is unused
```

**Review by jmartin-sul — CHANGES_REQUESTED** [review]
i think the rubocop exception comment needs to be re-added to keep the lint check from failing.

**Comment by ndushay** [comment]
@jmartin-sul after a bunch of digging, I figured out what was happening in rubocop and changed the syntax used so the rubocop exception is no longer needed.

image

**Comment by jmartin-sul** [comment]
> @jmartin-sul after a bunch of digging, I figured out what was happening in rubocop and changed the syntax used so the rubocop exception is no longer needed

nice, thanks for the explanation!

**Review by jmartin-sul — APPROVED** [review]

**Pull request merged**